backend/Pipfile

```diff
@@ -4,20 +4,21 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-black = "==19.10b0"
+black = "*"
```

Comment:
Why is this still here? And why is it = "*"? (Same for flake8)

Reply:
I might be mistaken, but I think the CI expects black and flake8 to be installed as part of the Django check, so until we migrate shared-actions to use ruff we should keep these here. I removed the version pins as I wasn't sure why they were necessary

Reply:
Just updated shared-actions to use ruff, and got rid of black, flake8, and isort